routes/controllers/yardimEt.js

```diff
@@ -142,7 +226,7 @@ module.exports = async function (fastifyInstance) {
       }
 
       const searchQuery = yardimTipi ? { yardimTipi } : {};
-
+      if ((hedefSehir && !iller.includes(hedefSehir)) || (sehir && !iller.includes(sehir))) return { "hata": "Lütfen geçerli bir şehir giriniz." }
```

Comment:
bunu JSON schema'da yapabilirsin. buraya eklemene gerek yok. ENUM value olarak arayabilirsin.

Reply:
az sonra yeni bir pr atacağım bunun ile alakalı, fix gerektiği için anlık yaptım. yoksa uzun uzadıya illeri dosyaya yazmak kirli duruyor evet.